godot/src/lib.rs

```diff
@@ -107,6 +107,13 @@
 //!   Use rustfmt to format generated binding code. Because rustfmt is so slow, this is detrimental to initial compile time.
 //!   Without it, we use a lightweight and fast custom formatter to enable basic human readability.
 //!
+//! * **`user-docgen`~~
+//!
+//!   Generates documentation for your structs from your rust documentation.
+//!   Documentation is visible in godot via `f1` -> searching for that class.
+//!   This feature requires an unstable version of godot, 4.3, so it requires the `api-custom` feature,
+//!   for the time being.<br><br>
```

Comment:
```suggestion
//!   Generates documentation for your structs from your Rust documentation.
//!   Documentation is visible in Godot via `F1` -> searching for that class.
//!   This feature requires at least Godot 4.3.
```

I would not mention the unstable because that's soon outdated, and there's already enough other docs to explain that.

Reply:
Maybe link to the `# Documentation` chapter in the macro? You can use a HTML path. We did this when linking from the `GodotClass` trait to the `#[derive(GodotClass)]` macro, so that could serve as inspiration.